code/modules/projectiles/guns/throw/crossbow.dm

```diff
@@ -1,26 +1,18 @@
-#define XBOW_TENSION_20 "20%"
-#define XBOW_TENSION_40 "40%"
-#define XBOW_TENSION_60 "60%"
-#define XBOW_TENSION_80 "80%"
-#define XBOW_TENSION_FULL "100%"
-
 /obj/item/gun/throw/crossbow
-	name = "powered crossbow"
-	desc = "A modern twist on an old classic. Pick up that can."
-	icon_state = "crossbow"
-	item_state = "crossbow-solid"
+	name = "Makeshift crossbow"
+	desc = "A Wood Cell Powered kind of Crossbow."
```

Comment:
Is it powered by a wood cell, or is it wooden and cell-powered? May I suggest `A wooden, cell-powered crossbow.` as the description?

Reply:
_I see Botany has been busy again_

Reply:
Wooden you know it 🤔

Reply:
Fixed , and yeah actually would be good if could be only loaded with potato batteries nice idea